task_12/index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" type="text/css" href="css/reset.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <title>Pizza App</title>
+  </head>
+  <body>
+    <header class="flex-container header">
+      <div class="lbl">
+        <i class="fa fa-clock-o fa-fw label" aria-hidden="true"></i>
+        <time id="clock">00:00:00</time>
+      </div>
+      <div class="flex-container center">
+        <img class="logo" src="img/pizza-cat.png" alt="logo">
+      </div>
+      <div class="lbl">
+        <i class="fa fa-user fa-fw label" aria-hidden="true"></i>
+        <a href="#">Login/Signup</a>
+      </div>
+    </header>
+    <main>
+      <div class="flex-container center">
+        <button class="btn" type="button">
+          <i class="fa fa-plus fa-fw label" aria-hidden="true"></i>
+          ADD NEW PIZZA
+        </button>
+      </div>
+      <div class="wrapper">
+        <div class="flex-container center padded position-1">
+          <div class="img-container">
+            <img class="tile" src="img/1.png" alt="image">
+          </div>
+          <div class="details">
+            <div class="flex-container">
+              <span>XX:XX:XX</span>
```

Comment:
These blocks for every item definitely require semantic element `<time>`.

Reply:
Also added.